tests/unit_tests/analysis/test_es_update.py

```diff
@@ -160,8 +160,8 @@ def test_that_posterior_has_lower_variance_than_prior(copy_case):
         (
             [
                 0.5895781800838542,
-                -0.732401196722254,
-                -1.0913320833716744,
+                -1.6225405348397028,
```

Comment:
why do we get different results?

Reply:
This is copied from the commit message:

Note that `projection` is now being passed to the smoothers.
This is because we only want to do projection when
the total number of parameters is less than ensemble_size - 1.
Because of this, the snapshot test_localization needs updating
as it ran using projection when it should not have.

So, pretty sure that the previous numbers were created with projection turned on when it should have been turned off.